### ISSUE TITLE
gernate-yaml seems broke

### ISSUE THREAD

**Issue by posix4e** [kubernetes-sigs/cluster-api-provider-ibmcloud#286]
/kind bug
/area provider/ibmcloud

**What steps did you take and what happened:**
[A clear and concise description of what the bug is.]
```
 $ ./generate-yaml.sh ./clouds.yaml ubuntu
no output folder provided, use name 'out' by default
cp: cannot stat '/home/posix4e/cluster-api-provider-ibmcloud/cmd/clusterctl/examples/ibmcloud/provider-component/clouds-secrets/configs/../../../../../../../vendor/sigs.k8s.io/cluster-api/config': No such file or directory
```

**What did you expect to happen:**
```
$ ./generate-yaml.sh ./clouds.yaml ubuntu
no output folder provided, use name 'out' by default
```

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]
Fixed it by going back to 66ae8f6f77291c7753d9ab8c0d4361612547eaf0

**Environment:** Linux

**Comment by xunpan** [comment]
@songleo 
please help to fix it. It seems introduced by PR#283.

**Comment by songleo** [comment]
@xunpan fixed in PR #289
